ui/app/components/secret-engine/configure-ssh.hbs

```diff
@@ -6,9 +6,9 @@
 <form {{on "submit" (perform this.save)}} aria-label="save ssh creds" class="has-top-padding-m" data-test-configure-form>
   <NamespaceReminder @mode="save" @noun="configuration" />
   <MessageError @errorMessage={{this.errorMessage}} />
-  {{#if @model.isNew}}
-    {{#each @model.formFields as |attr|}}
-      <FormField @attr={{attr}} @model={{@model}} @modelValidations={{this.modelValidations}} />
+  {{#if @configForm.isNew}}
```

Comment:
is `@configForm` the ssh config resource?

Reply:
No it's the form class. I just kept it consistent with the route model naming and the `configure-wif` component arg.

Reply:
Gotcha, thank you!